logger/logger.go

```diff
@@ -107,9 +110,6 @@ func Retrieve(filePath string) string {
 
 // FatalLog - exits os after logging
 func FatalLog(message ...string) {
-	var mu sync.Mutex
-	mu.Lock()
```

Comment:
should global lock remain?

Reply:
This one is not needed since it will cause an exit and does not actually add the to the logs in memory